spec/acceptance/letsencrypt_plugin_dns_rfc2136_spec.rb

```diff
@@ -26,24 +16,18 @@ class { 'letsencrypt::plugin::dns_rfc2136':
       }
     PUPPET
 
-    if supported
-      it 'installs letsencrypt and dns rfc2136 plugin without error' do
-        apply_manifest(pp, catch_failures: true)
-      end
-      it 'installs letsencrypt and dns rfc2136 idempotently' do
-        apply_manifest(pp, catch_changes: true)
-      end
+    it 'installs letsencrypt and dns rfc2136 plugin without error' do
+      apply_manifest(pp, catch_failures: true)
+    end
+    it 'installs letsencrypt and dns rfc2136 idempotently' do
+      apply_manifest(pp, catch_changes: true)
+    end
```

Comment:
Not for this PR, but this can be implemented via [a shared example](https://github.com/voxpupuli/voxpupuli-acceptance/#an-idempotent-resource)